libraries/SITL/SITL.cpp

```diff
@@ -1258,6 +1258,17 @@ const AP_Param::GroupInfo SIM::var_ins[] = {
     // @Description: If non-zero the vehicle will be clamped in position until the value on this servo channel passes 1800PWM
     AP_GROUPINFO("CLAMP_CH",     49, SIM, clamp_ch, 0),
 
+    // @Param: AIS_COUNT
+    // @DisplayName: Number of AIS vessels
+    // @Description: Total number of AIS simulated vessels
+    AP_GROUPINFO("AIS_COUNT", 50, SIM,  ais_vessel_count, -1),
```

Comment:
I would prefer the parameters to be in SIM_AIS.cpp

Reply:
The problem is that this serial driver lives in `AP_HAL_SITL`. So here in `SITL` we don't have a reference to get at the params by. 

I could still put them in there own table, but they I don't think they can go in the AIS class.

Reply:
Done, it is a little odd in that you have to set the serial port protocol and enable AIS so begin it called on the port and the simulator create and the params show up.